examples/rspack-minimal/rspack.config.js

```diff
@@ -91,6 +91,16 @@ const config = {
       disableClientServer: process.env.ENABLE_CLIENT_SERVER === 'false',
       features: ['bundle', 'plugins', 'loader'],
       mode: 'brief',
+      linter: {
```

Comment:
We should add a new E2E case instead of update the examples.

Reply:
i setup a new test to check linter rule render